File: src/Nethereum.Generators.JavaScript/codeGenNodeTests/MyContractName/CQS/TransferFunction.cs
using System;
using System.Threading.Tasks;
using System.Numerics;
using Nethereum.Hex.HexTypes;
using Nethereum.Contracts.CQS;
using Nethereum.ABI.FunctionEncoding.Attributes;
using StandardToken.MyContractName.DTOs;
namespace StandardToken.MyContractName.CQS
{
    [Function("transfer", "bool")]
    public class TransferFunction:ContractMessage
    {
        [Parameter("address", "_to", 1)]
        public string To {get; set;}
        [Parameter("uint256", "_value", 2)]
        public BigInteger Value {get; set;}
    }
}

File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/CQS/TransferFunction.vb
Imports System
Imports System.Threading.Tasks
Imports System.Collections.Generic
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.Contracts.CQS
Imports Nethereum.ABI.FunctionEncoding.Attributes
Imports StandardToken.MyContractName.DTOs
Namespace StandardToken.MyContractName.CQS

    <[Function]("transfer", "bool")>
    Public Class TransferFunction
        Inherits ContractMessage
    
        <[Parameter]("address", "_to", 1)>
        Public Property To As String
        <[Parameter]("uint256", "_value", 2)>
        Public Property Value As BigInteger
    
    End Class


End Namespace
